plugins/sync/src/test/kotlin/app/aaps/plugins/sync/garmin/LoopHubTest.kt

```diff
@@ -109,6 +113,22 @@ class LoopHubTest: TestBase() {
         verify(iobCobCalculator, times(1)).calculateIobFromBolus()
     }
 
+    @Test
+    fun testBasalOnBoard() {
+        val iobBasal = IobTotal(time = 0).apply { basaliob = 23.9 }
+        `when`(iobCobCalculator.calculateIobFromTempBasalsIncludingConvertedExtended()).thenReturn(iobBasal)
```

Comment:
nit: formatting?

Reply:
I don't see anything that is not formatted correctly. I have attached a screenshot because depending on the size of the section, the formatting may be displayed incorrectly. Or am I not understanding you correctly?
![image](https://github.com/nightscout/AndroidAPS/assets/12517693/9c22cc56-0b13-4337-8891-1d47df708317)

Reply:
Ah, I guess that was my problem. Thanks